client/trino-client/pom.xml

```diff
@@ -146,6 +146,14 @@
             <artifactId>junit-jupiter-api</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.arrow</groupId>
```

Comment:
unfortunately, this is incompatible with java 8. need to extract arrow decoding to a separate module